p2d/p2d.py

```diff
@@ -29,40 +28,45 @@ def main(args):
     def start(package_dir, output_dir, output_file):
         logger.info('This is p2d.py by cubercsl.')
         logger.info('Process Polygon Package to Domjudge Package.')
-        logger.info('Package directory: {}'.format(package_dir))
-        logger.info('Temp directory: {}'.format(output_dir))
-        logger.info('Output file: {}.zip'.format(output_file))
+        logger.info(f'Package directory: {package_dir}')
+        logger.info(f'Temp directory: {output_dir}')
+        logger.info(f'Output file: {output_file}.zip')
         input("Press enter to continue...")
         logger.info(END_OF_SUBPROCESS)
 
     def parse_problem():
         logger.info('Parse \'problem.xml\':')
-        xml_file = '{}/problem.xml'.format(package_dir)
+        xml_file = f'{package_dir}/problem.xml'
         root = xml.etree.ElementTree.parse(xml_file)
         name = root.find('names').find('name').attrib['value']
-        timelimit = float(math.ceil(float(root.find('judging').find('testset').find('time-limit').text) / 1000.0))
+        timelimit = int(root.find('judging').find('testset').find('time-limit').text) / 1000.0
+        memlimit = int(root.find('judging').find('testset').find('memory-limit').text) // 1024
```

Comment:
According to 

https://github.com/DOMjudge/domjudge/blob/acf58a7e3fe0c6c47c860490b280a42ba7ab76de/webapp/src/Service/ImportProblemService.php#L378

It should be megabytes.

This should be `// 1024 // 1024` or `// 1048576`